src/components/Sidebar/Sidebar.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../styles/Colors";
+@import "../../styles/Properties";
+
+.root {
```

Comment:
let's make sure we are using the colors we imported instead of hex values, and you shouldn't be needing to set font size or line height or anything. if you check the Components.scss under the styles/ folder, you should see that every type of h/p tag is defined for certain fonts/sizes.